formulaire/src/main/resources/public/ts/controllers/forms-list.ts

```diff
@@ -766,6 +767,12 @@ export const formsListController = ng.controller('FormsListController', ['$scope
         vm.limitTable += vm.tableSize;
     };
 
+    vm.isCloseConfirmationOpen = () : boolean => {
+        let elem = document.getElementsByTagName('share-panel')[0].firstElementChild;
+        let sharePanelScope = angular.element(elem).scope();
+        return sharePanelScope.display.showCloseConfirmation;
```

Comment:
rajouter un if si sharePanelScope et sharePanelScope.display existent

Reply:
Yes, tu peux risquer un cannot read property of undefined (si display est undefined ou son parent sharePanelScope)